node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/configserver/noderepository/Acl.java

```diff
@@ -21,16 +23,16 @@
  */
 public class Acl {
 
-    private final List<InetAddress> trustedNodes;
-    private final List<Integer> trustedPorts;
+    private final Set<Node> trustedNodes;
```

Comment:
Why is List changed to Set? List has the nice feature that order is preserved which is useful for humans (e.g. reading all config servers cfg1-3 close to each other), useful for testing that order is guaranteed, and may be important if a client makes IP table rules from these ACL lists.

Reply:
Discussed offline, but to sum up:
It feels wrong to rely that a method should return a specific order when the returned items don't have a natural ordering (i.e. the trusted nodes/ports are all equally trusted). When returning a set, it makes it clear to the user that there are no order guarantees and they should sort it however needed.